packages/styles/scss/patterns/sub-patterns/link-list/index.scss

```diff
@@ -0,0 +1,11 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '../../../globals/imports';
+@import '../../../globals/utils/hang';
+@import 'link-list';
+@import '@carbon/grid/scss/grid';
```

Comment:
@mkothur need to add card here too, right?

Reply:
> @mkothur need to add card here too, right?

Jeff later I looked in card is not needed I believe bcz CTA component is having customClass prop for heading. So, If I add card here in LinkList heading font color is becoming bold so.

Reply:
So if an adopter pulled in just the LinkList, and not pull in the Card scss, it would render as you would expect? What if an adopter pulled in the card scss separately, would that pose any issues?

Reply:
yes I guess if an adopter pull just LinkList is fine. If an adopter pull the card scss separately not cause any issue.

Reply:
It just seems odd to pull in a component without its core styles included. How is it working without the intended styles?

Reply:
never mind I interpreted wrongly here. We needed a Card scss but when I added that in LinkList index.scss, I am getting bold for resource heading. bcz CTA component has customClass of that.

Reply:
we'll need card.scss regardless so let's pull that in. Would we need override styles in this case for LinkList?

Reply:
yes we need to override that font style in LinkList.